sensor_msgs/PointCloud2 icp_cloud
int8[] index